<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayStatisticDataDialog</class>
 <widget class="QDialog" name="DisplayStatisticDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据统计</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../analysis_ready_data_tools.qrc">
    <normaloff>:/analysis_ready_data_tools/images/merge_ptp.png</normaloff>:/analysis_ready_data_tools/images/merge_ptp.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>356</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Button_Cancel">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Button_OK">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>543</width>
       <height>185</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_InputData">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>设置数据包路径</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_ClearDataPath">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>清空数据包路径列表</string>
           </property>
           <property name="text">
            <string>清空数据包路径列表</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>478</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_AddDataPath">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>添加数据包路径（文件夹）</string>
           </property>
           <property name="text">
            <string>添加数据包路径（文件夹）</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableWidget" name="tableWidget_InputDataPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>数据包路径</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>是否选择</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>是否通过下载方式获取</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>统计信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QTabWidget" name="tabWidget_DisplayStatisticData">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_TableDisplay">
            <attribute name="title">
             <string>表格展示</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableWidget_Display">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <column>
                <property name="text">
                 <string>数据包路径</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>数据包类型</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>数据量（GB）</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>条目数</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>下载量（GB）</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_HistogramDisplay">
            <attribute name="title">
             <string>柱状图展示</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_Chart"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../analysis_ready_data_tools.qrc"/>
 </resources>
 <connections/>
</ui>
